recipes-bsp/u-boot/u-boot-stm-common.inc

```diff
@@ -8,7 +8,6 @@ DEPENDS += "bison-native"
 
 SRC_URI = " \
 	git://gitlab.denx.de/u-boot/u-boot.git;protocol=https \
-	file://0001-remove-usb-power.patch \
```

Comment:
Yes, but CONFIG_ADC is enabled by default in stm32mp15_basic_defconfig, So, it is required to diable CONFIG_ADC.

Also, please drop this patch from your PR  and open a new one dedicated to U-Boot.

Reply:
Alright, but this patch broke the build, it is important to fix it quickly